eisenhower-matrix/src/main/webapp/index.html

```diff
@@ -18,8 +18,8 @@
     <nav id="header" class="navbar navbar-expand-lg navbar-light d-flex justify-content-between">
         <h1>Eisenhower Matrix Scheduler</h1>
       <div  class="flex-row-reverse" id="authentification">
-          <button id="logout" class="btn btn-default " onclick="logout()">Logout</button>
-          <button id="login" class="btn btn-default " onclick="login()">Login</button>
```

Comment:
Since the Calendar API doesnt use UserService as it's authentication, we probably don't need the login/logout functions in `script.js` and the `Login` servlet. For now, I left them in in case anyone was doing anything that depended on those services.